apps/vanilla/src/pages/blackjack/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./index.css" />
+    <title>Document</title>
+</head>
+
+<body class="default-layout">
+	<main>
+		<h1>Blackjack Dice</h1>
+		<div>
```

Comment:
```suggestion
		<section>
```
I think you can divide it into sections